packages/braintree-integration/src/braintree-local-payment-methods/braintree-local-methods-payment-strategy.ts

```diff
@@ -0,0 +1,237 @@
+import {
+    InvalidArgumentError,
+    MissingDataError,
+    MissingDataErrorType,
+    OrderFinalizationNotRequiredError,
+    OrderRequestBody,
+    PaymentArgumentInvalidError,
+    PaymentInitializeOptions,
+    PaymentIntegrationService,
+    PaymentMethodInvalidError,
+    PaymentRequestOptions,
+    PaymentStrategy,
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
+import BraintreeIntegrationService from '../braintree-integration-service';
+import {
+    BraintreeLocalMethods,
+    LocalPaymentInstance,
+    LocalPaymentsPayload,
+    onPaymentStartData,
+    StartPaymentError,
+    WithBraintreeLocalMethodsPaymentInitializeOptions,
+} from './braintree-local-methods-options';
+import { LoadingIndicator } from '@bigcommerce/checkout-sdk/ui';
+
+export default class BraintreeLocalMethodsPaymentStrategy implements PaymentStrategy {
+    private orderId?: string;
+    private localPaymentInstance?: LocalPaymentInstance;
+    private braintreeLocalMethods?: BraintreeLocalMethods;
+    private nonce?: string;
+    private loadingIndicatorContainer?: string;
+
+    constructor(
+        private paymentIntegrationService: PaymentIntegrationService,
+        private braintreeIntegrationService: BraintreeIntegrationService,
+        private loadingIndicator: LoadingIndicator,
```

Comment:
Do you really need `loadingIndicator`? Did you have a chance to check the flow without it?